File: rps.cxx
#include <iostream>
#include <random>
#include <cmath>
#include <math.h>
using namespace std;

int main(int argc, char *argv[])
{	
	int computerSeed;
	cout<<"Insert number seed for the computer rng\n";
	cin>>computerSeed;
	srand(computerSeed);
		
	while (true) {
		string playerChoice;
		cout<<"Choose: rock, paper, or scissors\n";
		cin>>playerChoice;
	
		int computerChoiceInt=rand()%3;
		
		string computerChoiceStr;
		switch (computerChoiceInt){
			case 0:
			computerChoiceStr="rock";
			break;
			case 1:
			computerChoiceStr="paper";
			break;
			case 2:
			computerChoiceStr="scissors";
			break;
			default:
			computerChoiceStr="broken";
			break;
			}		
		
		cout<<"\n\nPLAYER CHOICE: "<<playerChoice<<"\n";
		cout<<"COMPUTER CHOICE: "<<computerChoiceStr<<"\n\n";
		
		if (playerChoice==computerChoiceStr) {
			cout<<"Tie";
		}
		if (playerChoice=="rock"&&computerChoiceStr=="scissors") {
		cout<<"Player win";
		}
		if (playerChoice=="paper"&&computerChoiceStr=="rock") {
		cout<<"Player win";
		}
		if (playerChoice=="scissors"&&computerChoiceStr=="paper") {
		cout<<"Player win";
		}
		if (computerChoiceStr=="rock"&&playerChoice=="scissors") {
		cout<<"Computer win";
		}
		if (computerChoiceStr=="paper"&&playerChoice=="rock") {
			cout<<"Computer win";
		}
		if (computerChoiceStr=="scissors"&&playerChoice=="paper") {
		cout<<"Computer win";
		}
		
		cout<<"\n...\n";
	}	
}		

File: rps.py
import random
import time

while True:
	playerChoice = input("Rock, Paper, or Scissors? (all lowercase)")
	if playerChoice!="rock" and playerChoice!="paper" and playerChoice!="scissors":
		print("you picked incorrect")
	else:

		computerChoice = random.randint(1,3)
		if computerChoice==1:
			computerChoice="rock"
		if computerChoice==2:
			computerChoice="paper"
		if computerChoice==3:
			computerChoice="scissors"

		time.sleep(0.5)
		print("Player Chose: ",playerChoice)
		time.sleep(0.5)
		print("Computer Chose: ",computerChoice)
		time.sleep(0.5)

		if playerChoice==computerChoice:
			print("tie")
		if playerChoice=="rock" and computerChoice=="paper":
			print("lose")
		if playerChoice=="paper" and computerChoice=="scissors":
			print("lose")
		if playerChoice=="scissors" and computerChoice=="rock":
			print("lose")
		if playerChoice=="rock" and computerChoice=="scissors":
			print("win")
		if playerChoice=="paper" and computerChoice=="rock":
			print("win")
		if playerChoice=="scissors" and computerChoice=="paper":
			print("win")
	
		for x in range(0,3):
			time.sleep(0.5)
			print(".")